### ISSUE TITLE
How to migrate nested content

### ISSUE THREAD

**Issue by CimplexAB** [kgiszewski/Machina#3]
Hi! Could you please provide an example when migrating nested content?

> Migrate Nested Content
> machina-mnc
> 
> Requires -ncdtpa, and -udi Optionally -f, -p
> 
> Migrates Nested Content values to use UDI.

Lets say i have a document type called Slider (alias: slider) with a nested property type called Slides (alias: slides). Each slide is a document type with the name Slide (alias: slide) with the following properties: 
Name: Image, Alias: image
Name: Caption, Alias: caption

Now we want to fix the Image on each Slide, how should the command look like?

I guess something similar to this:
`machina-mnc -ncdtpa:image -f:slide -udi:media`

Thanks in advance!

**Comment by kgiszewski** [comment]
To be honest, I haven't used this in years.

I'd have to run the debugger and trace thru the code to answer your question. There's a good chance you'll have to create your own migration script to do what you are asking.

**Comment by CimplexAB** [comment]
Yeah i did :), your tool saved me tons of hours migrating to v8, thank you so much!
